examples/1_Introduction/05_usb_limited_angle_fbp_sirt.ipynb

```diff
@@ -505,12 +558,12 @@
     "plt.figure(figsize=(20,10))\n",
     "\n",
     "plt.subplot(1,2,1)\n",
-    "plt.plot(np.cos(np.deg2rad(data2d.geometry.angles)), np.sin(np.deg2rad(data2d.geometry.angles)),'.')\n",
```

Comment:
Had to make corrections such as this because the old TXRMReader used to save angles to the geometry as degrees whereas the new ZEISSReader saves them as radians, made this issue which relates to this: https://github.com/TomographicImaging/CIL/issues/1336